packages/@okta/vuepress-site/docs/reference/api/user-types/index.md

```diff
@@ -0,0 +1,374 @@
+
+##### Request Parameters
+
+The [User Type](#user-type-model) you wish to update
+
+##### Response parameters
+
+The updated [User types](#user-type-model)
+
+##### Request Example
+
+```bash
+curl -s -XPUT -H "Content-Type: application/json" -H "Authorization: SSWS ${api_token}" https://{yourOktaDomain}/api/v1/meta/types/user/otyfnly5cQjJT9PnR0g4 -d '{
+"description": "Updated description",
+"displayName": "Updated Name for UI",
+"name": "updatedTypeName"
+}'
+```
+
+##### Response Example
+
+```json
+{
+  "_links": {
+    "schema": {
+      "href": "https://{yourOktaDomain}/api/v1/meta/schemas/user/oscfnly5cQjJT9PnR0g4",
+      "method": "GET",
+      "rel": "schema"
+    },
+    "self": {
+      "href": "https://{yourOktaDomain}/api/v1/meta/types/user/otyfnly5cQjJT9PnR0g4",
+      "method": "GET",
+      "rel": "self"
+    }
+  },
+  "created": "2019-04-10T02:00:01.000Z",
+  "createdBy": "00ufnlhzppWItClAI0g4",
+  "default": false,
+  "description": "Any description that means something useful to you",
+  "displayName": "TheDisplayName",
+  "id": "otyfnly5cQjJT9PnR0g4",
+  "lastUpdated": "2019-04-10T02:00:01.000Z",
+  "lastUpdatedBy": "00ufnlhzppWItClAI0g4",
+  "name": "aNewType",
+  "ref": {
+    "id": "otyfnly5cQjJT9PnR0g4",
+    "refClass": "com.saasure.db.dto.platform.entity.cvd.CVDType"
+  }
+}
+```
+
+### Delete User Type
+
+<ApiOperation method="delete" url="/api/v1/meta/types/user/${typeId}" />
+
+Deletes a User Type permanently. This operation is not permitted for the default type, nor for any User Type that has existing users. After a User Type has been deleted, it cannot be used as the type for new users, and it no longer counts against the limit of 10 User Types.
+
+##### Request Parameters
+
+| Parameter | Description                                                                           | Param Type | DataType | Required | Default |
+| --------- | ------------------------------------------------------------------------------------- | ---------- | -------- | -------- | ------- |
+| typeId    | `id` of user type                                                                     | URL        | String   | TRUE     |         |
+
+##### Response Parameters
+
+None.
+
+Passing an invalid `typeId` returns a `404 Not Found` status code with error code `E0000007`.
+
+Passing the id of the default type returns a `403 Forbidden` status code with error code `E0000142` and a `reason` of `PROHIBITED`.
+
+Passing the id of a type for which non-deleted users exist returns a `403 Forbidden` status code with error code `E0000142` and a `reason` of `UNMET_REQUIREMENTS`.
+
+##### Request Example
+
+
+```bash
+curl -v -X DELETE \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://{yourOktaDomain}/api/v1/meta/types/user/otyfnly5cQjJT9PnR0g4"
+```
+
+##### Response Example
+
+
+```http
+HTTP/1.1 204 No Content
+```
+
+## Specify the User Type of a New User
```

Comment:
Please use sentence case for headings, so `Specify the User Type of a new user`

Reply:
Done.